README.md

```diff
@@ -37,7 +37,15 @@ This README will provide directions for building, testing, and debugging that co
             - **vfio** mechanism
             - **l2-controller** network service
             - **{ sriovToken: "l2.domain/1G" }** request parameters
-        
+* NSM_AWARENESS_GROUPS - Awareness groups for mutually aware NSEs
+* NSM_LIVENESSCHECKENABLED - Dataplane liveness check enabled/disabled
+* NSM_LIVENESSCHECKINTERVAL - Dataplane liveness check interval
+* NSM_LIVENESSCHECKTIMEOUT - Dataplane liveness check timeout
+* NSM_LOCALDNSSERVERADDRESS - Default address for local DNS server
+* NSM_LOCALDNSSERVERENABLED - Local DNS Server enabled/disabled
```

Comment:
As I can see, we also missed `split_words:"true"` in some environment names. 

I think it can also be improved and updated.

Then, for example, instead of `NSM_LIVENESSCHECKENABLED` we will use `NSM_LIVENESS_CHECK_ENABLED` that looks clearer.